src/refract/Registry.h

```diff
@@ -18,17 +18,21 @@ namespace refract
 {
     class Registry
     {
-        typedef std::map<std::string, std::unique_ptr<IElement> > Map;
-        Map registrated;
+    public:
+        using type_map = std::map<std::string, std::unique_ptr<IElement> >;
```

Comment:
Does this need to be public? If I understand correctly it's only used in Registry.cc as an implementation detail of Registry on how it is internally stored.

Reply:
It does not need to be public, but the implementation file can reference this type outside of member definitions if it is.